pom.xml

```diff
@@ -92,6 +92,17 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.20</version>
+                <configuration>
+                    <includes>
+                        <include>**/*Test.java</include>
+                        <include>**/*Spec.java</include>
+                    </includes>
```

Comment:
Fakt. Swoją drogą - przekopiowałem to z Waszego bloga i macie literówkę :)
https://touk.pl/blog/2013/08/08/spock-java-and-maven/
wszystkie otwierające tagi "groupId" i "artifactId" mają małe "i"